client/src/lib/votecounting.ts

```diff
@@ -91,10 +91,14 @@ export function* parseCVRs(
 
       for (const contestId in votes as VotesDict) {
         if (Object.prototype.hasOwnProperty.call(votes, contestId)) {
-          if (!ballotStyleContests.has(`${_ballotStyleId}/${contestId}`)) {
-            errors.push(
-              `Contest '${contestId}' in CVR is not in the election definition or is not a valid contest for ballot style '${_ballotStyleId}'`
-            )
+          // let's ignore any fields that start with '_' for some level of
+          // forwards-compatibility
+          if (!contestId.startsWith('_')) {
```

Comment:
This seems like excess checking. Shouldn't the schema checker take care of this?
https://github.com/votingworks/ballot-encoder/blob/main/src/schema.ts#L24

Reply:
this is *less* checking, not more, I'm explicitly allowing fields that start with an `_` but that are otherwise not recognized to go through. This is useful in one main way: future CVR formats can add extra `_` metadata fields without breaking deployed software.

Reply:
Ok, gotcha. Thanks for clarification.